{{ cookiecutter.format }}/app/build.gradle

```diff
@@ -22,8 +22,15 @@ android {
 
         python {
             version "{{ cookiecutter.python_version|py_tag }}"
+
+
             pip {
                 install "-r", "requirements.txt"
+
+                def f = new File(project.projectDir, "pip-options.txt")
+                def pipOptions = f.isFile() ? f.getText("utf-8").split("\n") : []
```

Comment:
Checking for the file's existence is unnecessary: it exists in the template, and Briefcase will never remove it, only overwrite it.